Targets/AppUIKit/Sources/wallet/WalletHomeView.swift

```diff
@@ -73,31 +105,31 @@ struct WalletHomeBody: View {
                     Spacer()
                 }
             }
-//            VStack {
-//                Spacer()
-//                Button{
-//                    path.append(Scanning(scanningType: .qrcode))
-//                } label: {
-//                    HStack(alignment: .center, spacing: 10) {
-//                        Image("QRCodeReader")
-//                            .resizable()
-//                            .frame(width: CGFloat(18), height: CGFloat(18))
-//                            .foregroundColor(.scanButton)
-//                        Text("Scan to share")
-//                            .font(.customFont(font: .inter, style: .medium, size: .h4))
-//                    }
-//                    .foregroundStyle(.white)
-//                    .padding(.vertical, 13)
-//                    .frame(width: UIScreen.screenWidth - 40)
-//                    .background(.scanButton)
-//                    .cornerRadius(100)
-//                    .overlay(
-//                        RoundedRectangle(cornerRadius: 100)
-//                            .stroke(.scanButton, lineWidth: 2)
-//                    )
-//                    .padding(.bottom, 6)
-//                }
-//            }
+            //            VStack {
+            //                Spacer()
+            //                Button{
+            //                    path.append(Scanning(scanningType: .qrcode))
+            //                } label: {
+            //                    HStack(alignment: .center, spacing: 10) {
+            //                        Image("QRCodeReader")
+            //                            .resizable()
+            //                            .frame(width: CGFloat(18), height: CGFloat(18))
+            //                            .foregroundColor(.scanButton)
+            //                        Text("Scan to share")
+            //                            .font(.customFont(font: .inter, style: .medium, size: .h4))
+            //                    }
+            //                    .foregroundStyle(.white)
+            //                    .padding(.vertical, 13)
+            //                    .frame(width: UIScreen.screenWidth - 40)
+            //                    .background(.scanButton)
+            //                    .cornerRadius(100)
+            //                    .overlay(
+            //                        RoundedRectangle(cornerRadius: 100)
+            //                            .stroke(.scanButton, lineWidth: 2)
+            //                    )
+            //                    .padding(.bottom, 6)
+            //                }
+            //            }
```

Comment:
```suggestion
```
I think we can remove this comment. It was basically an old "scan to share" button that I was keeping here.